ansible_collections/arista/avd/molecule/eos_designs_deprecated_vars/inventory/host_vars/host1/name_servers.yml

```diff
@@ -0,0 +1,4 @@
+# name_servers is deprecated in 4.5.0. To be removed in 5.0.0
+name_servers:
+  - 192.168.200.5
```

Comment:
config is not generated should we not expect "some config change" for the negative tests as it was not tested before?

Reply:
Good catch. The scenario was not included in MOLECULE_SCENARIOS or CI. I will add it in this PR.

Reply:
https://github.com/aristanetworks/ansible-avd/pull/3283/commits/af9822f9d0a9652eceacc29a80e52a74f823ad51